extensions.json

```
@@ -999,6 +999,10 @@
       "checkout": "v0.27.0",
       "location": "packages/vscode-ruby-client"
     },
+    {
+      "id": "reduckted.vscode-gitweblinks",
+      "repository": "https://github.com/reduckted/vscode-gitweblinks"
```

Since they're available, I think we should checkout and pin a specific tag/version for this extension:

```suggestion
      "repository": "https://github.com/reduckted/vscode-gitweblinks",
      "version": "1.9.0",
      "checkout": "v1.9.0"
```